content/blog/2019/11/2019-11-21-welcome-to-the-matrix.adoc

```diff
@@ -0,0 +1,389 @@
+---
+layout: post
+title: "Welcome to the Matrix"
+tags:
+- pipeline
+- plugins
+- declarative
+- matrix 
+author: lnewman
+---
+
+I often find myself needing to run the same actions on a bunch of different configurations.  
+Up to now, that meant I had to make multiple copies of the same stages in my pipelines. 
+When I needed to make changes, I had to make the same changes in multple places throughout my pipepine.
```

Comment:
When I needed to make changes, I had to make the same changes in multiple places throughout my pipeline.

Reply:
@richbg
Thanks for the review. Have you seen the "Suggestion" feature?  It really smoothes out the process
<img width="330" alt="Screen Shot 2019-11-21 at 11 26 23 AM" src="https://user-images.githubusercontent.com/1958953/69369945-c9751800-0c51-11ea-8845-f340329f83f2.png">